noir-projects/aztec-nr/aztec/src/state_vars/shared_mutable/scheduled_delay_change.nr

```diff
@@ -142,7 +144,7 @@ impl<let INITIAL_DELAY: u32> Packable<1> for ScheduledDelayChange<INITIAL_DELAY>
         [packed.to_integer()]
     }
 
-    fn unpack(input: [Field; 1]) -> Self {
+    fn unpack(input: [Field; SCHEDULED_DELAY_CHANGE_PCKD_LEN]) -> Self {
```

Comment:
Insted of defining packable for svc and sdc, we can create the struct that contains both and just implement packable for that. If we do this, we can combine sdc and svc, since svc has a block of change (32 bits) that can be merged into sdc. Then we'd have 2*N+1 total instead of 2*N+2, which makes public writes cheaper and private reads a bit cheaper (less hashing).

We can do this in a separate PR.

Reply:
If we wanted to be really nasty we can have a manual `storage_read` where we read a single field at the base slot plus offset of the field in which we combine these, and then manually unpack just that field. We do save a storaage write with this approach.

Reply:
I don't understand